dashboard/app/models/script.rb

```diff
@@ -1623,4 +1626,16 @@ def self.has_any_pilot_access?(user = nil)
     return true if user.permission?(UserPermission::LEVELBUILDER)
     all_scripts.any? {|script| script.has_pilot_experiment?(user)}
   end
+
+  def csf?
+    return true if curriculum_umbrella == 'CSF'
+  end
+
+  def csd?
+    return true if curriculum_umbrella == 'CSD'
+  end
+
+  def csp?
+    return true if curriculum_umbrella == 'CSP'
```

Comment:
these helper methods are a great touch!

Reply:
Turns out we already have a helper `csf?` https://github.com/code-dot-org/code-dot-org/blob/529afde8ab4379b9fb2865efe5d27ac50badd3c3/dashboard/app/models/script.rb#L689, which means that `k5_course?` is yet another place where we have a hardcoded list of of csf versions and another candidate for cleaning up with the new property. I'm going to remove the helpers for now, and add them in a later PR where they will be used and not overwrite the current method.